### PR TITLE
Introduce the concept of playbooks early on

### PR DESCRIPTION
**Pull request by lbt**
##### SUMMARY
I began learning about ansible yesterday and was rather confused about playbooks.
There is no initial concept of what a simple playbook looks like, where to store it or how to run it - so the subsequent documentation has no foundation for the reader to build upon.

Given the section above encourages the user to run a live command I thought it appropriate to run a trivial playbook

This also teaches that ansible does not manage playbook storage for you (I spent quite some time wondering about that!) and how to execute them.

Finally, as a puppet user, it was getting pretty frustrating to not see "real stuff" happening and running the playbook was a nice "ah ha!" moment for me.

Now I have a baseline to hack on and try out things in the docs :)

##### ISSUE TYPE
- Docs Pull Request

+label: docsite_pr

##### ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- Test Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: docs/docsite/rst/user_guide/intro_getting_started.rst [hunk 1]
```diff
@@ -94,6 +94,56 @@ You should see output for each host in your inventory, similar to this:
     aserver.example.org | CHANGED | rc=0 >>
     hello
 
+Action: Run your first playbook
+-------------------------------
+
+Playbooks are used to pull together tasks into reusable units.
+
+Ansible does not store playbooks for you; they are simply yaml documents that you store and manage, passing them to Ansible to run as needed.
```
Comment (samccann) [inline docs/docsite/rst/user_guide/intro_getting_started.rst]:
```suggestion
Ansible does not store playbooks for you; they are simply YAML documents that you store and manage, passing them to Ansible to run as needed.
```

File: docs/docsite/rst/user_guide/intro_getting_started.rst [hunk 2]
```diff
@@ -94,6 +94,56 @@ You should see output for each host in your inventory, similar to this:
...
+
+In a directory of your choice you can create your first playbook by creating a file called mytask.yml (the extension is optional but may help ediors assist you with formatting). Enter:
```
Comment (samccann) [inline docs/docsite/rst/user_guide/intro_getting_started.rst]:
```suggestion
In a directory of your choice you can create your first playbook in a file called mytask.yml:
```

File: docs/docsite/rst/user_guide/intro_getting_started.rst [hunk 3]
```diff
@@ -94,6 +94,56 @@ You should see output for each host in your inventory, similar to this:
...
+.. code-block:: yaml
+
+    ---
+    - name: My task
+      hosts: all
+      tasks:
+         - name: Leaving a mark
+           command: "touch /tmp/ansible_was_here"
+
+You would run this like so:
```
Comment (samccann) [inline docs/docsite/rst/user_guide/intro_getting_started.rst]:
```suggestion
You can run this command as follows:
```

File: docs/docsite/rst/user_guide/intro_getting_started.rst [hunk 4]
```diff
@@ -94,6 +94,56 @@ You should see output for each host in your inventory, similar to this:
...
+   insufficient you can add 'warn: false' to this command task or set 'command_warnings=False' in ansible.cfg to get rid of this message.
+   changed: [aserver.example.org]
+   changed: [bserver.example.org]
+   
+   PLAY RECAP ******************************************************************************************************************************
+   aserver.example.org        : ok=2    changed=1    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0   
+   bserver.example.org        : ok=2    changed=1    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0   
+   192.0.2.50                 : ok=0    changed=0    unreachable=1    failed=0    skipped=0    rescued=0    ignored=0   
+
+Read on to learn more about controlling which nodes playbooks are executed on, more sophisticated tasks and the meaning of the output.
```
Comment (samccann) [inline docs/docsite/rst/user_guide/intro_getting_started.rst]:
```suggestion
Read on to learn more about controlling which nodes your playbooks execute on, more sophisticated tasks, and the meaning of the output.
```

**Review by samccann — COMMENTED** [review]
Thanks this is a great addition to the getting started docs! I've made a few suggested edits before we merge.

File: docs/docsite/rst/user_guide/intro_getting_started.rst (hunk 4 above)
Comment (lbt) [reply docs/docsite/rst/user_guide/intro_getting_started.rst]: That looks good to me.

**Review by samccann — APPROVED** [review]

**Comment by samccann** [comment]
Thanks @lbt for the Ansible docs improvement!

**Pull request merged**